src/components/CtaSection.tsx

```diff
@@ -148,7 +160,7 @@ export const CtaSection: FunctionComponent<CtaSection> = ({
 
             <div
                 className={classNames({
-                    'tw-inline-flex tw-flex-col tw-self-center': centerContent,
+                    'tw-inline-flex tw-self-center': centerContent,
```

Comment:
On mobile, the buttons need to be stacked, and then after the `xs` breakpoint, we can set them to be side by side with the right margin to space them out.

Reply:
One of these pages had them side by side so we had some contradicting styles between pages. That style sounds fine to me 👍🏻

Reply:
Cool no worries! I think whenever we discover discrepancies in other pages, let's flag them to Tracey so we can gain some insight for the specs and then match the DLS more closely, or Tracey can revise the DLS to match the suggested specs.